pages/management/applications.md

```diff
@@ -52,13 +52,13 @@ __Warning:__ This action is only supported on devices with an Agent version >= 1
 
 ### Reboot All Devices
 
-This action allows you to perform a reboot of all the devices in the fleet/application. This is different from the `Restart Application` action mentioned above, because in this action, the entire device including the kernel will be rebooted as if there was a power cycle. It should be noted that currently these action notifications are not queued up, so if a device is offline when the action is triggered, it will never be notified of the action it missed.
+This action allows you to perform a reboot of all the devices in the fleet/application. This is different from the `Restart Application` action mentioned above, because in this action, the entire device including the kernel will be rebooted as if you were to manually replug it. It should be noted that currently these action notifications are not queued up, so if a device is offline when the action is triggered, it will never be notified of the action it missed.
```

Comment:
IMO `power cycle` is a well-known term our users are familiar with, so there's no need to replace it